addons/sale_timesheet/tests/test_project_update.py

```diff
@@ -9,6 +9,10 @@
 @tagged('-at_install', 'post_install')
 class TestProjectUpdateSaleTimesheet(TestProjectUpdate):
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
I don't think you need to override it, do you? It will get called anyway I think